lib/utils/exec.go

```diff
@@ -80,6 +80,11 @@ func RunPlanetCommand(ctx context.Context, log log.FieldLogger, args ...string)
 	return RunCommand(ctx, log, args...)
 }
 
+// RunInPlanetCommand executes the command specified with args inside planet container
+func RunInPlanetCommand(ctx context.Context, log log.FieldLogger, args ...string) ([]byte, error) {
```

Comment:
But you already introduced `PlanetEnterCommand`?

Reply:
`PlanetEnterCommand` builds arguments for the command. This thing runs it.